app/controllers/users_controller.rb

```diff
@@ -25,6 +25,6 @@ def update
   private
 
   def user_params
-    params.require(:user).permit(:is_beta_tester, :vita_partner_id, :timezone)
+    params.require(:user).permit(*(:is_super_admin if current_user.is_super_admin?), :is_beta_tester, :vita_partner_id, :timezone)
```

Comment:
`(*(:is_super_admin if current_user.is_super_admin?)...`

Needlessly clever? Probably.

Reply:
never seen the syntax but assuming that it doesn't pass in any argument if `current_user.is_super_admin?` is false?

Reply:
```
puts(*(:is_super_admin if true), :is_beta_tester, :vita_partner_id, :timezone)
irb(main):001:0> puts(*(:is_super_admin if true), :is_beta_tester, :vita_partner_id, :timezone)
is_super_admin
is_beta_tester
vita_partner_id
timezone
=> nil


irb(main):003:0> puts(*(:is_super_admin if false), :is_beta_tester, :vita_partner_id, :timezone)
is_beta_tester
vita_partner_id
timezone
=> nil


irb(main):004:0> puts(*(:is_super_admin if nil), :is_beta_tester, :vita_partner_id, :timezone)
is_beta_tester
vita_partner_id
timezone
=> nil
```